src/langs/en.json

```diff
@@ -76,8 +77,9 @@
     "Click here if you are not automatically redirected": "Click here if you are not automatically redirected",
     "Create Shortcut": "Create Shortcut",
     "Sign Out": "Sign Out",
-    "Confirm deleting item.": "Confirm deleting item",
-    "This item will be deleted permanently.": "This item will be deleted permanently.",
+    "Confirm deletion": "Confirm deletion",
+    "itemDeleteMessage_one": "This item will be deleted permanently.",
+    "itemDeleteMessage_other": "These items will be deleted permanently.",
```

Comment:
Mhmmm so how does i18n do it ? You have a suffix that is dependant on the quantity ?

Reply:
There's a mechanism that automatically match the `count` parameter to `_one` or `_other` (and more). So it chooses the right translation!